apps/site/src/app/admin/applicants/[uid]/components/ApplicationReviews.tsx

```diff
@@ -0,0 +1,43 @@
+import { useContext } from "react";
+
+import ApplicantStatus from "@/app/admin/applicants/components/ApplicantStatus";
+import { Review, uid } from "@/lib/admin/useApplicant";
+import UserContext from "@/lib/admin/UserContext";
+
+interface ApplicationReviewsProps {
+	reviews: Review[];
+}
+
+function ApplicationReviews({ reviews }: ApplicationReviewsProps) {
+	const { uid } = useContext(UserContext);
+
+	if (reviews.length === 0) {
+		return <p>-</p>;
+	}
+
+	const formatUid = (uid: uid) => uid.split(".").at(-1);
+	const formatDate = (timestamp: string) =>
+		new Date(timestamp).toLocaleDateString();
```

Comment:
Note for later: Same as with submission, it might also be good to include the time a review was made instead of just the date.